test/checks/livenessProbeMissing/success.yaml

```diff
@@ -6,16 +6,17 @@ metadata:
   name: liveness-exec
 spec:
   containers:
-  - name: liveness
-    image: k8s.gcr.io/busybox
-    args:
+  - args:
     - /bin/sh
     - -c
     - touch /tmp/healthy; sleep 30; rm -rf /tmp/healthy; sleep 600
-    livenessProbe:
+    image: k8s.gcr.io/busybox
+    livenessProbe: #TODO: Change livenessProbe setting to reflect your health endpoints
```

Comment:
I think we should avoid re-using the success files as mutation results - it could get confusing.

Can we revert the success.yaml files and create mutated.yaml files instead?

Reply:
done